Networking/Networking/Model/Refund/OrderItemRefund.swift

```diff
@@ -0,0 +1,163 @@
+        let productID = try container.decode(Int.self, forKey: .productID)
+        let variationID = try container.decode(Int.self, forKey: .variationID)
+        let decimalQuantity = try container.decode(Decimal.self, forKey: .quantity)
+        let quantity = NSDecimalNumber(decimal: decimalQuantity)
+        let decimalPrice = try container.decodeIfPresent(Decimal.self, forKey: .price) ?? Decimal(0)
+        let price = NSDecimalNumber(decimal: decimalPrice)
+        let sku = try container.decodeIfPresent(String.self, forKey: .sku)
+        let subtotal = try container.decode(String.self, forKey: .subtotal)
+        let subtotalTax = try container.decode(String.self, forKey: .subtotalTax)
+        let taxClass = try container.decode(String.self, forKey: .taxClass)
+        let taxes = try container.decode([OrderItemTaxRefund].self, forKey: .taxes)
+        let refundTotal = try container.decode(String.self, forKey: .refundTotal)
+        let totalTax = try container.decode(String.self, forKey: .totalTax)
+
+        // initialize the struct
+        self.init(itemID: itemID, name: name, productID: productID, variationID: variationID, quantity: quantity, price: price, sku: sku, subtotal: subtotal, subtotalTax: subtotalTax, taxClass: taxClass, taxes: taxes, refundTotal: refundTotal, totalTax: totalTax)
+    }
+
+    /// The public encoder for OrderItemRefund.
+    ///
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: EncodingKeys.self)
+
+        try container.encode(itemID, forKey: .itemID)
+        try container.encode(name, forKey: .name)
+        try container.encode(productID, forKey: .productID)
+        try container.encode(variationID, forKey: .variationID)
+        try container.encode(Double(truncating: quantity), forKey: .quantity)
+        try container.encode(price.stringValue, forKey: .price)
+        try container.encode(sku, forKey: .sku)
+        try container.encode(subtotal, forKey: .subtotal)
+        try container.encode(subtotalTax, forKey: .subtotalTax)
+        try container.encode(taxClass, forKey: .taxClass)
+
+        try container.encode(taxes, forKey: .taxes)
+
+        try container.encode(refundTotal, forKey: .refundTotal)
+        try container.encode(totalTax, forKey: .totalTax)
+    }
+}
+
+
+/// Defines all of the OrderItemRefund CodingKeys.
+///
+private extension OrderItemRefund {
+
+    enum EncodingKeys: String, CodingKey {
+        case itemID         = "id"
+        case variationID    = "variation_id"
+        case name
+        case productID      = "product_id"
+        case quantity
+        case price
+        case sku
+        case subtotal
+        case subtotalTax    = "subtotal_tax"
+        case taxClass       = "tax_class"
+        case refundTax      = "refund_tax"
+        case refundTotal    = "refund_total"
+        case totalTax       = "total_tax"
+        case taxes
+    }
+
+    enum DecodingKeys: String, CodingKey {
```

Comment:
I reckon it makes sense to declare encoding and decoding keys separately, as we might not want to encode and decode the same properties. But it seems like there are properties like SKU that we are not going to be able to mutate locally, so would it make sense to use a single enum to declare the keys?